scripts/common/preflights.sh

```diff
@@ -899,3 +900,20 @@ function bail_if_kurl_version_is_lower_than_previous_config() {
         log "and the current kURL version used is $KURL_VERSION"
     fi
 }
+
+# bail if rook/openebs prereqs not met for minimumNodeCount param for automated storage scaling
+function bail_if_automated_storage_scaling_prereqs_not_met() {
+    if [ -n "ROOK_MINIMUM_NODE_COUNT"]; then
+        semverCompare "$(echo "$ROOK_VERSION" | sed 's/v//g')" "$(echo "1.11.5" | sed 's/v//g')"
+        if [ "$SEMVER_COMPARE_RESULT"  = "-1" ]; then # less than or equal to 1.11.5
+            logFail "The current Rook version $ROOK_VERSION is less than 1.11.5 which is required to use automated storage scaling (minimumNodeCount Parameter)"
+            bail "Please use a Rook version higher than 1.11.5 or remove minimumNodeCount Paramenter"
+        fi
+        semverCompare "$(echo "$OPENEBS_VERSION" | sed 's/v//g')" "$(echo "3.6.0" | sed 's/v//g')"
```

Comment:
Per my previous [comment](https://github.com/replicatedhq/kURL/pull/4659/files#r1246573460) for Rook we also need to settle on the minimum version for OpenEBS.